core/pom.xml

```diff
@@ -170,6 +170,14 @@
           <target>1.7</target>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>8</source>
+          <target>8</target>
```

Comment:
maven-compiler-plugin is already added just above this...why is this required?

Reply:
fixed